src/EA.Iws.Web.Tests.Unit/Controllers/AdminExportNotificationMovements/CaptureMovementControllerTests.cs

```diff
@@ -61,25 +62,25 @@ public CaptureMovementControllerTests()
             RouteAssert.RoutesTo(result.RouteValues, "Index", "InternalCapture", "AdminExportMovement");
         }
 
-        [Fact]
-        public void Create_Get_RetrievesMovementNumber()
-        {
-            controller.TempData["MovementNumberKey"] = 52;
+        //[Fact]
```

Comment:
If these tests are no longer needed they should be deleted rather than commented out.